js/apps/system/_admin/aardvark/APP/react/package.json

```diff
@@ -169,5 +170,8 @@
     "presets": [
       "react-app"
     ]
+  },
+  "resolutions": {
+    "ansi-html": "https://registry.yarnpkg.com/ansi-html-community/-/ansi-html-community-0.0.8.tgz#69fbc4d6ccbe383f9736934ae34c3f8290f1bf41"
```

Comment:
Not sure if the full URL is really needed. Since the package exists at https://www.npmjs.com/package/ansi-html-community, it should suffice to just map to that package by name and version.

See https://github.com/yarnpkg/rfcs/blob/master/implemented/0000-selective-versions-resolutions.md#mapping-version-specifications-as-well-as-packages-name

Reply:
Thanks, let me check.

Reply:
I tried and I failed, not working with that much cleaner and simpler solution. So unfortunately, we've to stick to this.

Reply:
Ok, since this is the only option that works.